src/ring_theory/polynomial/hermite/gaussian.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2023 Luke Mantle. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Luke Mantle, Jake Levinson
+-/
+
+import ring_theory.polynomial.hermite.basic
+import analysis.special_functions.exp
+import analysis.special_functions.exp_deriv
+
+/-!
+# Hermite polynomials and Gaussians
+
+This file shows that the Hermite polynomial `hermite n` is (up to sign) the
+polynomial factor occurring in the `n`th derivative of a gaussian.
+
+## Results
+
+* `polynomial.hermite_eq_deriv_gaussian`: the Hermite polynomial is (up to sign) the polynomial
+factor occurring in the `n`th derivative of a gaussian.
+
+## References
+
+* [Hermite Polynomials](https://en.wikipedia.org/wiki/Hermite_polynomials)
+
+-/
+
+noncomputable theory
+open polynomial
+
+namespace polynomial
+
+/- `hermite n` is (up to sign) the factor appearing in `deriv^[n]` of a gaussian -/
+lemma hermite_eq_deriv_gaussian (n : ℕ) (x : ℝ) :
+  aeval x (hermite n) =
+  (-1 : ℝ)^n * (deriv^[n] (λ y, real.exp (-(y^2 / 2))) x) / real.exp (-(x^2 / 2)) :=
```

Comment:
Is this also true as the simpler
```suggestion
  aeval (-x) (hermite n) =
    (deriv^[n] (λ y, real.exp (-(y^2 / 2))) x) / real.exp (-(x^2 / 2)) :=
```

Reply:
I think this is a better version:
```suggestion
  aeval x (hermite n) =
    (-1 : ℝ)^n * (deriv^[n] (λ y, real.exp (-(y^2 / 2))) x) * real.exp (x^2 / 2) :=
```
I don't see why having `-x` on the LHS is a good idea, this makes applying this lemma worse.

Reply:
I don't agree with that version being better, it no longer contains `gaussian x` as a repeated subexpression

Reply:
`gaussian x` isn't a thing yet in Lean, right? Should it be (with its own little api, simp-lemmas, etc) ?

Reply:
No and I think it only makes sense to have that as an element in `schwartz_map`. For the bare function it is overkill.

Reply:
@jcommelin @mcdoll These (the `*` vs `/` versions of the lemma, and whether to define `gaussian x`) were discussed further up the PR in a now-resolved conversation.

Reply:
I think the original version was just fine and in the discussion I see no good argument for changing to the current version. If we really cared about readability of proofs, we had to rewrite almost all of mathlib.

Reply:
The current version now includes both versions of the statement.

Reply:
(And if possible I'd like to not make further substantial changes)